manifests/r/REDModding/WolvenKit/Nightly/8.8.1-nightly.2023-01-14/REDModding.WolvenKit.Nightly.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: REDModding.WolvenKit.Nightly
+PackageVersion: 8.8.1-nightly.2023-01-14
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
+UpgradeBehavior: install
+ReleaseDate: 2023-01-14
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
Installers:
```

Reply:
The package installs normally but presents this error when launched:
![image](https://user-images.githubusercontent.com/114952565/212797559-6f27c949-9f07-476a-8a01-52b4f6dfe0a4.png)